Documentation/faq.md

```diff
@@ -1,5 +1,11 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, general
+
+#### Does the clients need to send requests to the leader?
+
+[Raft][raft] is leader-based, and the leader handles all client requests. Client does not need to know which node is the leader. Request to follower is automatically forwarded to leader. Raft algorithm divides time into terms, and each term begins with an election to choose one leader. If leader becomes unavailable, cluster immediately elects a new leader or automatically transfers the leadership to a follower. This requires no human intervention.
```

Comment:
I wonder whether is it necessary to include how raft does election in this question. Suppose that I know raft, then i know how election works. If I don't know raft, i probably won't understand the election strategy even after reading the answer. My main question is that how detail should we answer the faq?